plugins/module_utils/network/ios/rm_templates/evpn_global.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+"""
+The Evpn_global parser templates file. This contains
+a list of parser definitions and associated functions that
+facilitates both facts gathering and native command generation for
+the given network resource.
+"""
+
+import re
+
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.rm_base.network_template import (
+    NetworkTemplate,
+)
+
+
+class Evpn_globalTemplate(NetworkTemplate):
+    def __init__(self, lines=None, module=None):
+        super(Evpn_globalTemplate, self).__init__(lines=lines, tmplt=self, module=module)
+
+    # fmt: off
+    PARSERS = [
+        {
+            "name": "default_gateway.advertise",
+            "getval": re.compile(
+                r"""
+                \sdefault-gateway\sadvertise
+                $""",
+                re.VERBOSE,
+            ),
+            "setval": "default-gateway advertise",
+            "result": {"default_gateway": {"advertise": True}},
+            "shared": True,
```

Comment:
```suggestion
```
might not be needed required here, only required when we want to share result data with other parsers